src/mgos_imu_mpu6886.c

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2018 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "mgos_imu_mpu6886.h"
+#include "mgos_imu_mpu60x0.h"
+#include "mgos.h"
+#include "mgos_i2c.h"
+
+static bool mgos_imu_mpu6886_detect(struct mgos_i2c *i2c, uint8_t i2caddr) {
+  if (!i2c) {
+    return false;
+  }
+
```

Comment:
Considering the experimental nature of the driver (re-using the 60x0 code and having known issues) can you emit an WARNING log line here saying something like "This driver is experimental and work in progress. Proceed with caution!" ?

Reply:
added